src/main/java/com/boomi/flow/services/boomi/mdh/records/GoldenRecordQueryRequest.java

```diff
@@ -15,6 +15,8 @@
     private View view;
     private Sort sort;
     private Filter filter;
+    private String offsetToken;
+    private String limit;
```

Comment:
Same as above, could this be a number/int instead of a string?

Reply:
I have changed to integer those values